LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWCovertActionsModTemplate.uc

```diff
@@ -28,22 +28,24 @@ static function UpdateCovertActions(X2StrategyElementTemplate Template, int Diff
 	
 	switch (CATemplate.DataName)
 	{
+		case 'CovertAction_SuperiorWeaponUpgrade':
+		case 'CovertAction_SuperiorPCS':
+		CaTemplate.RequiredFactionInfluence=eFactionInfluence_Minimal;
 		case 'CovertAction_GatherSupplies':
 		case 'CovertAction_GatherIntel':
 		case 'CovertAction_ImproveComInt':
 		case 'CovertAction_FormSoldierBond':
-		case 'CovertAction_SharedAbilityPoints':
-		case 'CovertAction_SuperiorWeaponUpgrade':
-		case 'CovertAction_SuperiorPCS':
 		case 'CovertAction_AlienLoot':
 			ConfigureEasyCovertAction(CATemplate);
 			break;
 		case 'CovertAction_RecruitScientist':
 		case 'CovertAction_RecruitEngineer':
+		CaTemplate.RequiredFactionInfluence=eFactionInfluence_Respected;
```

Comment:
It doesn't really mitigate much because it's as inconsistent as getting one from missions. Black market already does a decent job at mitigating early sci/engineers. plus those two are significantly stronger than other basic covert actions, making people keen to restart untill they got those, or complain when they don't.

Reply:
I think the re-roll is mostly a problem with the first covert action having no risks associated with it. Otherwise, as I said, the costs (opportunity and direct) make for an interesting decision, and provide an alternative for when you don't have any supplies for scientists/engineers.

As for mitigation, having two sources of probability means an increased chance overall for a mission/CA to get one of those units.

Reply:
> Otherwise, as I said, the costs (opportunity and direct) make for an interesting decision
it's simply much stronger CA at that point in the game than any other available, regardless of your early luck with missions. It's just as likely to give you the CA while you'll have good luck with missions, simply giving you too many of them as a result. It should be treated on par with the recruit additional hero class CA.